src/main/java/com/fasterxml/jackson/dataformat/xml/deser/XmlTokenStream.java

```diff
@@ -559,6 +561,9 @@ private final String _collectUntilTag() throws XMLStreamException
             if (FromXmlParser.Feature.EMPTY_ELEMENT_AS_NULL.enabledIn(_formatFeatures)) {
                 return null;
             }
+            if (FromXmlParser.Feature.EMPTY_ELEMENT_AS_EMPTY_ARRAY.enabledIn(_formatFeatures)) {
+                return JsonToken.START_ARRAY.asString() + JsonToken.END_ARRAY.asString();
```

Comment:
I am not sure how this makes sense -- all we get here is String "[]", which won't be further decoded?